praw/models/reddit/live.py

```diff
@@ -319,6 +320,32 @@ def contributor(self) -> LiveContributorRelationship:
         """
         return LiveContributorRelationship(self)
 
+    @cachedproperty
+    def stream(self) -> "LiveThreadStream":
+        """Provide an instance of :class:`.LiveThreadStream`.
+
+        Streams are used to indefinitely retrieve new updates made to a
+        live thread, like:
+
+        .. code-block:: python
+
+           for live_update in reddit.live("ta535s1hq2je").stream.updates():
+               print(live_update.body)
```

Comment:
Could you make this indented 4 spaces instead of 3? I plan on standardizing this at some point. 
```suggestion
            for live_update in reddit.live("ta535s1hq2je").stream.updates():
                print(live_update.body)
```